src/components/config-pages/global-message.tsx

```diff
@@ -24,38 +22,30 @@ const GlobalMessage = ({
   });
 
   return (
-    <div className={wrapperClasses + " py-10"}>
-      <div className="centered flex flex-col lg:flex-row gap-10">
-        <div className="flex items-center leading-none shrink-0">
-          <MessageIcon messageType={suGlobalMsgType}/>
-          {suGlobalMsgLabel}:
```

Comment:
No more label?

Reply:
I made the poor assumption that since we were not using the icons that we would not need the labels. I'll triple check with our designers and update this based on their response 😄

Reply:
I've confirmed with Ali that we would like to keep the icon + label but only have it display if the user provides a label 👍 PR has been updated to reflect this change.